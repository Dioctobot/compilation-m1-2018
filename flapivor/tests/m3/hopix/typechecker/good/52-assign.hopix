val x : Ref<Int> = ref (0)

val main : Unit =
    x := 1