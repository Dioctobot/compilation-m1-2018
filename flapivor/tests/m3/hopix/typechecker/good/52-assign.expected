x : Ref<Int>
main : Unit
